association_backend/association/domain/entities/subscription_entities.py

```diff
@@ -0,0 +1,85 @@
+from dataclasses import dataclass
+from datetime import datetime
+from enum import Enum
+from typing import Optional
+
+import pydantic
+from sqlalchemy import Column, DateTime, ForeignKey, Integer, String, Table
+from sqlalchemy.orm import registry, relationship
+
+
+class UserData(pydantic.BaseModel):
+    email: str
+    user_id: int
```

Comment:
I don't think we should have this, as you will receive a `Pastaporto` object from the gateway

Reply:
I don't know, it's a support entity without database persistence.
It is currently used to pass data to services and I would not delete it.
If we switch to management without one of the 2 fields, we will certainly be able to eliminate it